core/sail/fts/elasticsearch/pom.xml

```diff
@@ -15,6 +15,8 @@
 
 	<properties>
 		<elasticsearch.version>1.7.0</elasticsearch.version>
+                <log4j.version>2.6.2</log4j.version>
+                <fasterxml.jackson.version>2.8.3</fasterxml.jackson.version>
```

Comment:
Can't upgrade a 3rd party dependency without filing a CQ and getting legal approval first.